logbook/logbook.go

```diff
@@ -1018,22 +1018,20 @@ func (book *Book) ConstructDatasetLog(ctx context.Context, ref dsref.Ref, histor
 	return book.save(ctx)
 }
 
-func itemFromOp(ref dsref.Ref, op oplog.Op) DatasetLogItem {
-	return DatasetLogItem{
-		VersionInfo: dsref.VersionInfo{
-			Username:   ref.Username,
-			ProfileID:  ref.ProfileID,
-			Name:       ref.Name,
-			Path:       op.Ref,
-			CommitTime: time.Unix(0, op.Timestamp),
-			BodySize:   int(op.Size),
-		},
+func versionInfoFromOp(ref dsref.Ref, op oplog.Op) dsref.VersionInfo {
+	return dsref.VersionInfo{
+		Username:    ref.Username,
+		ProfileID:   ref.ProfileID,
+		Name:        ref.Name,
+		Path:        op.Ref,
+		CommitTime:  time.Unix(0, op.Timestamp),
+		BodySize:    int(op.Size),
 		CommitTitle: op.Note,
```

Comment:
Is logbook also going to be adding RunID here? I'm assuming it will, but that it requires work in some other PR.

Reply:
yep!